constants/extraRpcs.js

```diff
@@ -49,6 +49,8 @@ const privacyStatement = {
     "Usage Data is collected automatically when using the Service. Usage Data may include information such as Your Device's Internet Protocol address (e.g., IP address), browser type, browser version, the pages of our Service that You visit, the time and date of Your visit, the time spent on those pages, unique device identifiers and other diagnostic data. When You access the Service by or through a mobile device, we may collect certain information automatically, including, but not limited to, the type of mobile device You use, Your mobile device unique ID, the IP address of Your mobile device, Your mobile operating system, the type of mobile Internet browser You use, unique device identifiers and other diagnostic data. We may also collect information that Your browser sends whenever You visit our Service or when You access the Service by or through a mobile device. https://www.liveplex.io/privacypolicy.html",
   jellypool:
     "The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analyzing trends, administering the site, tracking users' movement on the website, and gathering demographic information. https://www.jellypool.xyz/privacy/",
+  rpcfast:
+    "RPC Fast API is designed to prioritize user privacy, and therefore, we do not collect any personal information when clients interact with the blockchain through our provider. We do not track IP addresses, wallet addresses, executed transactions, or any other identifying data during interactions with our provider. For more info: https://rpcfast.com/privacy"
```

Comment:
the privacy policy at https://rpcfast.com/privacy doesn't match whats on this line, instead it says
> We collect personally-identifying information that clients provide when subscribing to or operating on our Services.